adage/analyze/management/commands/import_node_gene_network.py

```diff
@@ -123,17 +141,26 @@ def check_and_import(file_handle, ml_model):
                 continue
 
             # Raise an exception if the combination of (node, gene)
-            # already exists in Particapation table.  Instead of relying
+            # already exists in Participation table. Instead of relying
             # on the IntegrityError exception implicitly, we raise an
             # explicit exception that includes the input file's line
             # number where the error is detected, and node name and gene
             # name involved.
+            # * Note: We do not need to check if the Participation's
+            # participation_type matches with our participation_type when
+            # we check if the Participation obj already exists,
+            # since a unique node AND a unique gene should yield at most
+            # *one* Participation object.
```

Comment:
Ah ok, it's true, it is possible for one same gene and node to have different participation types, so I will change that here and in the ``unique_together`` restriction in ``models.py``. Good catch!